generic/nbody/handlers-mutation.links

```diff
@@ -0,0 +1,389 @@
+# =================================================================
```

Comment:
I suggest create a directory `handlers/` in the root, and move this file there. Rename the file to `nbody.links`